index.js

```diff
@@ -31,14 +31,14 @@ class Analytics {
    *   @property {Number} [retryCount] (default: 3)
    */
 
-  constructor (writeKey, options) {
```

Comment:
Let's consider avoiding any such small changes like this, in order to make following upstream as easy as possible.

Reply:
yup some autoformat stuff. Removed